drivers/dma/dma_sam0.c

```diff
@@ -415,7 +415,9 @@ static int dma_sam0_init(const struct device *dev)
 
 	/* Reset the DMA controller */
 	DMAC->CTRL.bit.DMAENABLE = 0;
-	DMAC->CTRL.bit.CRCENABLE = 0;
+#ifdef DMAC_CTRL_CRCENABLE
+		DMAC->CTRL.bit.CRCENABLE = 0;
```

Comment:
Do not add additional indentation for `ifdef` blocks.

```suggestion
	DMAC->CTRL.bit.CRCENABLE = 0;
```